decentraland/unity-renderer
# fix: footsteps heard on moving platforms

## What this PR includes?

Fixes #289 



## How to test it?

1. Go to: https://play.decentraland.zone/index.html?ENV=org&renderer=urn:decentraland:off-chain:renderer-artifacts:fix/footsteps-heard-on-moving-platforms

2. Go to any moving platform and stand still. Note that footsteps are not heard anymore when the platform moves.
